jekyll/_posts/2018-10-01-hacktoberfest-with-nim.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Hacktoberfest with Nim
+author: The Nim Team
```

Comment:
Feel free to include your name here next time :)

I'll keep it as is in case you don't want your name here.